doc/content/verification_and_validation/ver-1kb.md

```diff
@@ -1,39 +1,83 @@
 
 ## Case Set Up
 
-This verification problem is taken from [!cite](ambrosek2008verification). 
+This verification problem is taken from [!cite](ambrosek2008verification).
 
-Over time, the pressures of T$_2$, which diffuses across the membrane in accordance with Henry’s law, will gradually equilibrate between the two enclosures. 
+Over time, the pressures of T$_2$, which diffuses across the membrane in accordance with Henry’s law, will gradually equilibrate between the two enclosures.
+
+The diffusion process in each of the two enclosures can be described by the following equations:
+
+\begin{equation}
+\frac{\partial C_1}{\partial t} = D \nabla^2 C_1
+\end{equation}
+
+\begin{equation}
+\frac{\partial C_1}{\partial t} = D \nabla^2 C_1
+\end{equation}
+
+where $C_1$, $C_2$ represent the concentration fields in enclosures 1 and 2 respectively, and $D$ denotes the diffusivity.
 
 The concentration in Enclosure 1 is related to the partial pressure and concentration in Enclosure 2 via the interface sorption law:
 
 \begin{equation}
-C_s = K P^n = K \left( \frac{C_g RT}{n} \right)
+C_1 = K P_2^n = K \left( \frac{C_2 RT}{n} \right)
 \end{equation}
 
 where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature in K, $K$ is the solubility, and $n$ is the exponent of the sorption law. For the Henry’s law, $n=1$.
 
-
 ## Results
 
-The TMAP8 pressure evolutions in the two enclosures are shown in [ver-1kb_comparison_time] as a function of time. 
+Two subcases are considered. In the first subcase, it is assumed that the pressures in the two enclosures are in equilibrium, implying $K = 1/RT$.
+Consistent with the results from TMAP7, the pressure evolution in both enclosures is shown in [ver-1kb_comparison_time] as a function of time, confirming the equilibrium between the pressures in enclosures 1 and 2. The linear regression in [ver-1kb_comparison_concentration] demonstrates that the concentration values at the interface comply with the sorption law, with a proportionality coefficient consistent with the solubility value $K = \approx 2.4 \times 10^{-4}$. As shown in [ver-1kb_mass_conservation], mass is conserved between the two enclosures over time.
 
-!media comparison_ver-1kb.py 
+!media comparison_ver-1kb.py
        image_name=ver-1kb_comparison_time.png
        style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
        id=ver-1kb_comparison_time
-       caption=Equilibration of species pressures under Henry’s law as predicted by TMAP8.
+       caption=Equilibration of species pressures under Henry’s law for $K=1/RT$.
+
+!media comparison_ver-1kb.py
+       image_name=ver-1kb_comparison_concentration.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1kb_comparison_concentration
+       caption=Concentration in enclosure 1 as a function of pressure in enclosure 2 at the interface for $K=1/RT$. Validation of the sorption law across the interface.
```

Comment:
```suggestion
       caption=Concentration in enclosure 1 as a function of pressure in enclosure 2 at the interface for $K=1/RT$. This verifies TMAP8's ability to apply the sorption law across the interface without a concentration jump.
```

Validation requires comparison against experimental data, verification is comparison against analytical solution. This here is a verification, not a validation.